pom.xml

```diff
@@ -54,7 +54,7 @@
       - Version properties must be sorted alphabetically (other form of sorting were found to be unclear and ambiguous).
     -->
     <version.aopalliance>1.0</version.aopalliance>
-    <version.asm>3.3.1</version.asm>
+    <version.asm>7.1</version.asm>
```

Comment:
This version doesn't exist: after `3.3.1` they changed the groupdId to `org.ow2.asm` and the corresponding property is `version.org.ow2.asm` that is already `7.1`

Btw we should probably review the usages of `asm:asm:3.3.1` and try to migrate to the newer version but this is a comment for another ticket :)